frb_codegen/src/ir/ty_optional_list.rs

```diff
@@ -0,0 +1,46 @@
+use crate::ir::*;
+use crate::target::Target;
+
+crate::ir! {
+    #[no_serde]
+    pub struct IrTypeOptionalList {
```

Comment:
nit: it would be great to mention briefly why we implement this Vec-Option, instead of using the already existing Vec.

Reply:
Good call, I'll add some notes in the docs somewhere.